hologram_dashboard/README.md

```diff
@@ -0,0 +1,12 @@
+## Server 
+Follow the following steps to run the server 
+    1.  install bazel following the steps of 
```

Comment:
Please make the instruction more concrete, for example,

1. git clone ...
2. cd ???
3. install bazel
4. ...